package.json

```diff
@@ -25,29 +25,31 @@
     "url": "https://github.com/my_name/jupyterlab_apod.git"
   },
   "scripts": {
-    "build": "jlpm run build:lib",
+    "build": "jlpm run build:lib && jlpm run build:labextension",
+    "build:all": "jlpm run build:lib && jlpm run build:labextension",
     "build:labextension": "jupyter labextension build .",
+    "build:labextension:dev": "jupyter labextension build --development True .",
     "build:lib": "tsc",
-    "build:all": "jlpm run build:lib && jlpm run build:labextension",
     "clean": "jlpm run clean:lib",
-    "clean:lib": "rimraf lib tsconfig.tsbuildinfo",
-    "clean:labextension": "rimraf jupyterlab_apod/static",
     "clean:all": "jlpm run clean:lib && jlpm run clean:labextension",
+    "clean:labextension": "rimraf jupyterlab_apod/static",
+    "clean:lib": "rimraf lib tsconfig.tsbuildinfo",
     "eslint": "eslint . --ext .ts,.tsx --fix",
     "eslint:check": "eslint . --ext .ts,.tsx",
+    "install:extension": "jupyter labextension develop --overwrite .",
     "prepare": "jlpm run clean && jlpm run build",
     "watch": "run-p watch:src watch:labextension",
-    "watch:src": "tsc -w",
-    "watch:labextension": "jupyter labextension watch ."
+    "watch:labextension": "jupyter labextension watch .",
+    "watch:src": "tsc -w"
   },
   "dependencies": {
-    "@jupyterlab/application": "^3.0.0-alpha.14",
-    "@jupyterlab/apputils": "^3.0.0-alpha.14",
+    "@jupyterlab/application": "^3.0.0-rc.5",
+    "@jupyterlab/apputils": "^3.0.0-rc.5",
     "@lumino/messaging": "^1.4.3",
     "@lumino/widgets": "^1.14.0"
   },
   "devDependencies": {
-    "@jupyterlab/builder": "^3.0.0-alpha.14",
+    "@jupyterlab/builder": "^3.0.0-rc.2",
```

Comment:
Interestingly the upgrade script did not update this one all the way up to `^3.0.0-rc.5`.

Reply:
Manually updated in https://github.com/jupyterlab/jupyterlab_apod/pull/8/commits/a93463d6aafb252beed5bbe5837a0c5cf0f0cedb for now.